docs/k8s_deploy.md

```diff
@@ -71,20 +61,17 @@ Destroy the container: `helm del --purge cnf1`
 ## Overview setup and steps
 
 VLAN assignments
-quad port intel:
-  vlan1 => eth1 (req)
-  vlan2 => eth2 (req)
-  vlan3 => eth3 (cluster mgmt)
+* quad port intel: 
+  * vlan1 => eth1 (req)
+  * vlan2 => eth2 (req)
+  * vlan3 => eth3 (cluster mgmt)
+* dual port Mellanox:
+  * vlan1 => eth1
+  * vlan2 => eth1
 
-dual port Mellanox:
-   vlan1 => eth1
-   vlan2 => eth1
 
-
-1. cross-cloud (terraform)
-2. terraform-ansible runs the playbook k8s_cluster.yml 
-3. k8s_cluster.yml playbook include playbooks to setup k8s cluster
-4. quad_intel_workers.yml (set interfaces var to Packet interfaces)
+1. cross-cloud container creates hosts via terraform and deploys kubernetes
+2. when VPP is enabled (defualt) ansible-playbook runs the playbook specified in the .env file (default: k8s_worker_vswitch_mellanox.yml) to setup the k8s cluster L2 networking
```

Comment:
The layer-2 networking is setup even if a vSwitch is not deployed.  Please add a step about that.

Reply:
It may be a misunderstanding on my part but I took a look into the cross-cloud code base [here](https://github.com/crosscloudci/cross-cloud) and couldn't find any ansible call that executed the TF_VAR_playbook being passed into it in the current setup. Hence, from what I can tell, the layer 2 playbooks are only run when the VPP switch is enabled:
https://github.com/cncf/cnf-testbed/blob/af872311e6ca8f8758d01f2fdb744afcfb4506f6/tools/deploy_k8s_cluster.sh#L17-L58

If layer 2 is being configured by cross cloud in another way, could you direct me to that so I may document it thoroughly?